File: api/src/main/java/net/funnyguilds/drama/config/DiscordConfig.java
package net.funnyguilds.drama.config;

import eu.okaeri.configs.OkaeriConfig;
import eu.okaeri.configs.annotation.NameModifier;
import eu.okaeri.configs.annotation.NameStrategy;
import eu.okaeri.configs.annotation.Names;
import lombok.Getter;
import lombok.Setter;

import java.util.Collections;
import java.util.List;

@Getter
@Setter
@Names(strategy = NameStrategy.HYPHEN_CASE, modifier = NameModifier.TO_LOWER_CASE)
public class DiscordConfig extends OkaeriConfig {
    private List<String> midnightDramaHooks = Collections.singletonList("https://discord.com/api/webhooks/x/y");
}

File: api/src/main/java/net/funnyguilds/drama/config/DiscordConfig.kt
package net.funnyguilds.drama.config

import eu.okaeri.configs.OkaeriConfig
import eu.okaeri.configs.annotation.NameModifier
import eu.okaeri.configs.annotation.NameStrategy
import eu.okaeri.configs.annotation.Names

@Names(strategy = NameStrategy.HYPHEN_CASE, modifier = NameModifier.TO_LOWER_CASE)
class DiscordConfig : OkaeriConfig() {
    var midnightDramaHooks = listOf("https://discord.com/api/webhooks/x/y")
}
